inc/composer/class-command.php

```diff
@@ -68,7 +70,7 @@ protected function configure() {
 	import-uploads                Copies files from `content/uploads` to s3
 EOT
 			)
-			->addOption( 'xdebug' );
+			->addOption( 'xdebug', null, InputOption::VALUE_OPTIONAL, 'Start the server with XDebug', 'debug' );
```

Comment:
Shouldn't this use `Xdebug` instead of `XDebug`, too?

```suggestion
			->addOption( 'xdebug', null, InputOption::VALUE_OPTIONAL, 'Start the server with Xdebug', 'debug' );
```

Reply:
Ah nice one. I realised I'd been writing it wrong all this time!

Reply:
I had to look it up because I assumed it was XDebug, too 😜